frame/bridge/subswap/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+authors     = ["Darwinia Network <hello@darwinia.network>"]
+description = "Interact with Sub Swap"
+edition     = "2018"
+homepage    = "https://darwinia.network/"
+license     = "GPL-3.0"
+name        = "darwinia-subswap"
```

Comment:
This pallet should not be called as `subswap`, subswap is a thirdparty brand not owned by Darwinia.

To simplify this, I think we can use a smart contract as a helper to execute the operations. This pallet only need to support transfer some balances to a DVM Account, which as my understanding, not even need a pallet.

cc @xiaoch05 @hujw77

Reply:
The contract does all the things, such as add and removes liquidity, send ERC20 tokens or native tokens. And the owner of this contract is System account `0x0`. I think the ERC20 is not saved in the pallet, instead, it's saved in this contract.